src/data/events.ts

```diff
@@ -11,19 +11,50 @@ import type { GetImageResult } from 'astro';
 import * as qs from 'qs';
 import type { Event as EventCMS } from './payload-types.ts';
 
-export interface Event {
+export interface EventCalendarEntry {
   title: string;
   date: Date;
   description: string;
-  content_html?: string;
   link?: string;
-  slug?: string;
   image: {
     src: Awaited<ReturnType<typeof getImage>>;
   };
 }
 
-export async function getNext3Events(): Promise<[Event, Event, Event]> {
+export interface EventPage {
+  title: string;
+  start: Date;
+  end: Date;
+  slug: string;
+  content_html: string;
+  heroImage?: {
+    src: Awaited<ReturnType<typeof getImage>>;
```

Comment:
We use this cumbersome type in quite a few places. maybe we should define it somewhere as `OptimizedImage` or something?

Reply:
Take a look at this:
```ts
import type { GetImageResult } from 'astro';
```